src/lib/browse/colors.ts

```diff
@@ -18,7 +18,8 @@ export const colors = {
   bus_route_with_lane: "#9362BA",
   bus_route_without_lane: "#C2A6D8",
   cycle_parking: "black",
-  crossings: "green",
+  signalized_crossing: "green",
```

Comment:
I'm just making colors up :(

We can display arbitrary icons in maplibre pretty easily; if we found a good set with a compatible license (like the ones from an existing OSM web map), we could consider using those in the future